src/Modal/__mocks__/mockAboutModalManager.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { Button, Modal } from '../../index'
+import CustomModalDialog from '../CustomModalDialog'
+
+export class MockAboutModalManager extends React.Component {
+  constructor() {
+    super()
+    this.state = { showModal: false }
+    this.open = this.open.bind(this)
+    this.close = this.close.bind(this)
+  }
+  open() {
+    this.setState({ showModal: true })
+  }
+  close() {
+    this.setState({ showModal: false })
+  }
+  render() {
+    return (
+      <div>
```

Comment:
Since react-16 you don't need to wrap everything in one div anymore:
https://reactjs.org/blog/2017/09/26/react-v16.0.html#new-render-return-types-fragments-and-strings

tbh, I'm not sure yet if it is a better approach.

Reply:
@sharvit are you using react-16 yet in Foreman? It would be great to get some indication when we  can safely target react-16 as there is a lot of new features i'd love to introduce (returning arrays is one of them).

Reply:
Yes we do!

Reply:
sooo @sharvit ... it looks React 16.2 adds support for fragments:
https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html

My vote would be to adopt that syntax now if we can... 🏃 🏃 🏃

Reply:
@priley86 I love the new syntax and would like to adopt it 👍

Reply:
@sharvit this is going to have to wait until #87 lands

Reply:
@sharvit @jgiardino it looks like we can extend `withInfo` method with some options...
https://github.com/storybooks/storybook/tree/master/addons/info#basic-usage

I've updated the PR to do this. Let me know if this approach works for now or you'd like to change the sample JSX shown. I'm not sure about how we could go about adding the PropTypes/Docgen comments in these kind of stories, so I'm going to remove them for this example for now.